src/Octoshift/Services/ReclaimService.cs

```diff
@@ -202,6 +212,27 @@ private bool HandleResult(string mannequinUser, string targetUser, Mannequin man
             return false;
         }
 
+        _log.LogInformation($"Reclaimation email successfully sent to: {mannequinUser} ({mannequin.Id}) for {targetUser} ({targetUserId})");
+
+        return true;
+    }
+
+    private bool HandleReclaimationResult(string mannequinUser, string targetUser, Mannequin mannequin, string targetUserId, MannequinReclaimResult result)
+    {
+        if (result.Errors != null)
+        {
+            _log.LogError($"Failed to reclaim {mannequinUser} ({mannequin.Id}) to {targetUser} ({targetUserId}) Reason: {result.Errors[0].Message}");
+            return false;
+        }
+
+        if (result.Data.ReattributeMannequinToUserInput is null ||
```

Comment:
What is the situation where this would happen?

Reply:
It would have to be some sort of failed hand-off with the API. Honestly, I pattern matched from the previous function (now `HandleInvitaitonResult`) to create this since I wasn't sure exactly what I could expect in return from the API.